<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>H Lab WebCam</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/icon.png</normaloff>:/img/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionCapture"/>
   <addaction name="actionFull"/>
   <addaction name="actionConfig"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConfig">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/settings.png</normaloff>:/img/settings.png</iconset>
   </property>
   <property name="text">
    <string>Config</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/download.png</normaloff>:/img/download.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/download_cancel.png</normaloff>:/img/download_cancel.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionCapture">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/camera.png</normaloff>:/img/camera.png</iconset>
   </property>
   <property name="text">
    <string>capture</string>
   </property>
  </action>
  <action name="actionFull">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/fullscreen.png</normaloff>:/img/fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>full</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
